code/game/objects/items/weapons/kitchen.dm

```diff
@@ -17,34 +17,31 @@
 	origin_tech = "materials=1"
 
 
-
-
 /*
  * Utensils
  */
 /obj/item/kitchen/utensil
-	force = 5.0
+	force = 5
 	w_class = WEIGHT_CLASS_TINY
-	throwforce = 0.0
 	throw_speed = 3
 	throw_range = 5
 	flags = CONDUCT
 	attack_verb = list("attacked", "stabbed", "poked")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
-	sharp = 0
+	sharp = TRUE
 	var/max_contents = 1
 
-/obj/item/kitchen/utensil/New()
-	..()
+/obj/item/kitchen/utensil/Initialize(mapload)
+	. = ..()
 	if(prob(60))
-		src.pixel_y = rand(0, 4)
-
+		pixel_y = rand(0, 4)
 	create_reagents(5)
 
-/obj/item/kitchen/utensil/attack(mob/living/carbon/C, mob/living/carbon/user)
-	if(!istype(C))
+/obj/item/kitchen/utensil/attack(mob/living/M, mob/living/carbon/user, def_zone)
+	if(!iscarbon(M))
 		return ..()
+	var/mob/living/carbon/C = M
```

Comment:
Why change the target argument from carbon to living? It saves having to cast it later on line 44.

Reply:
Yeah, this is bad form; it was better the way it was, before.

Reply:
I was under the impression that the way I changed it to was preferred, since the parent proc has `mob/living` as an argument too rather than `mob/living/carbon`. Much easier to read in my opinion.

Reply:
That's a fair point, keeping it consistent with the parent args, not sure the style preferences on that. As for this type of thing being done elsewhere, it is done fairly often. By that I mean having the exact desired type in the argument, and then doing an `istype(X)`.